crates/extensions/c8y_config_manager/src/actor.rs

```diff
@@ -433,6 +417,21 @@ impl ConfigManagerMessageBox {
     }
 }
 
+#[async_trait]
+impl ReceiveMessages<ConfigInput> for ConfigManagerMessageBox {
+    async fn try_recv(&mut self) -> Result<Option<ConfigInput>, RuntimeRequest> {
+        self.input_receiver.try_recv().await
+    }
+
+    async fn recv_message(&mut self) -> Option<WrappedInput<ConfigInput>> {
+        self.input_receiver.recv_message().await
+    }
+
+    async fn recv(&mut self) -> Option<ConfigInput> {
+        self.input_receiver.recv().await
+    }
+}
+
```

Comment:
Having to repeat ourselves, here, is a sign that this actor should use a `SimpleMessageBox`.